gef-ui/server/controllers/application-details/index.js

```diff
@@ -102,6 +103,7 @@ function buildBody(app, previewMode, user) {
           facilityName: item.details.name,
           // ukefFacilityId required for html facility summary table id
           ukefFacilityId: item.details.ukefFacilityId,
+          stage: item.details?.facilityStage ?? (item.details.hasBeenIssued ? STAGE.ISSUED : STAGE.UNISSUED),
```

Comment:
Are we able to create a helper for this? Looks like the same/similar logic is duplicated in `display-items.js`

Reply:
The logic is slightly different in each place unfortunately - here `hasBeenIssued` is a boolean whereas in `display-items.js`,  `hasBeenIssued` is a string, so we can't comfortably use the same function for each. I'm happy to keep as is to reduce extra files for using in one place. What do you think?